pkg/lib/k8s/ingress.go

```diff
@@ -1,174 +0,0 @@
-/*
```

Comment:
I think we can leave this file even if we don't use it right now, since it's in `lib`